tests/wpt/tests/dom/events/listener-adopt-from-iframe.tentative.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta name="author" title="Josh Matthews" href="mailto:josh@joshmatthews.net">
+<link rel="help" href="https://github.com/servo/servo/issues/36397">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe></iframe>
+<script>
+  async_test(t => {
+    window.imageLoaded = false;
+    var iframe = document.querySelector('iframe');
+    iframe.onload = function() {
+      let img = iframe.contentDocument.body.firstChild;
+      document.body.appendChild(img);
+      iframe.remove();
+      img.addEventListener('load', () => {
+        assert_equals(window.imageLoaded, false, "Window should point to iframe contentWindow");
```

Comment:
The message here is confusing, since it seems to contradict what the test is checking. Doesn't the iframe content window have imageLoaded=true because the element's onload handler executes before the iframe's load event is fired?